src/main/java/stream/flarebot/flarebot/mod/modlog/ModAction.java

```diff
@@ -0,0 +1,42 @@
+package stream.flarebot.flarebot.mod.modlog;
+
+public enum ModAction {
+
+    BAN(true, ModlogEvent.USER_BANNED),
+    FORCE_BAN(true, ModlogEvent.USER_BANNED),
+    TEMP_BAN(true, ModlogEvent.USER_TEMP_BANNED),
+    UNBAN(false, ModlogEvent.USER_UNBANNED),
+
+    KICK(true, ModlogEvent.USER_KICKED),
+
+    TEMP_MUTE(true, ModlogEvent.USER_TEMP_MUTED),
+    MUTE(true, ModlogEvent.USER_TEMP_MUTED),
+    UNMUTE(false, ModlogEvent.USER_TEMP_MUTED),
+
+    WARN(true, ModlogEvent.USER_TEMP_MUTED);
```

Comment:
Copy and pasting is doing well here :wink: Make sure the events are the actual thing you're doing

Reply:
Shhh